src/main/webapp/app/entities/course/course.component.html

```diff
@@ -1,13 +1,21 @@
 <div>
-    <h4>
-        <span id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses</span>
-        <button *jhiHasAnyAuthority="'ROLE_ADMIN'" class="btn btn-primary float-right jh-create-entity create-course" [routerLink]="['/course/new']">
-            <fa-icon [icon]="'plus'"></fa-icon>
-            <span class="hidden-sm-down" jhiTranslate="artemisApp.course.home.createLabel">
-                Create new Course
-            </span>
-        </button>
-    </h4>
+    <div class="d-flex">
+        <h4 id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses</h4>
+        <div class="ml-auto">
+            <div class="btn-group btn-group-toggle">
+                <label class="btn-primary" ngbButtonLabel>
+                    <input type="checkbox" formControlName="toggleOnlyActive" ngbButton (change)="showOnlyActive = !showOnlyActive" />
```

Comment:
This causes an error in console for me because the formControl is not embedded in a form.
You can just remove the `formControlName` here.

```
breadcrumbs.js?fd98:58 ERROR Error: formControlName must be used with a parent formGroup directive.  You'll want to add a formGroup
       directive and pass it an existing FormGroup instance (you can create one in your class).

      Example:

      
    <div [formGroup]="myGroup">
      <input formControlName="firstName">
    </div>

    In your class:

    this.myGroup = new FormGroup({
       firstName: new FormControl()
    });
```

Reply:
As I now have a checkbox instead of a toggle button and I removed the formControlName, this should no longer be a problem